src/main/webapp/app/exam/manage/student-exams/student-exam-detail.component.ts

```diff
@@ -107,8 +108,29 @@ export class StudentExamDetailComponent implements OnInit {
         const workingTime = this.artemisDurationFromSecondsPipe.transform(this.studentExam.workingTime);
         const workingTimeParts = workingTime.split(':');
         this.workingTimeForm = new FormGroup({
-            minutes: new FormControl(parseInt(workingTimeParts[0] ? workingTimeParts[0] : '0', 10), [Validators.min(0), Validators.required]),
-            seconds: new FormControl(parseInt(workingTimeParts[1] ? workingTimeParts[1] : '0', 10), [Validators.min(0), Validators.max(59), Validators.required]),
+            minutes: new FormControl({ value: parseInt(workingTimeParts[0] ? workingTimeParts[0] : '0', 10), disabled: this.examIsVisible() }, [
+                Validators.min(0),
+                Validators.required,
+            ]),
+            seconds: new FormControl({ value: parseInt(workingTimeParts[1] ? workingTimeParts[1] : '0', 10), disabled: this.examIsVisible() }, [
+                Validators.min(0),
+                Validators.max(59),
+                Validators.required,
+            ]),
         });
     }
+
+    examIsVisible(): boolean {
+        if (this.studentExam.exam) {
+            return moment(this.studentExam.exam.visibleDate).isBefore(moment());
+        }
+        // if the exam is not visible to students yet, we disable the form to edit the working time
```

Comment:
I don't get this comment. For two reasons:
- The purpose of the PR is to ensure that the working time cannot be changed after the exam is visible to the students. If there is no visibleDate the working time can be changed --> so the method should return false (--> form is editable)
- The first part (*if the exam is not visible to students yet*) is a little confusing. If the visibleDate is not set would make more sense here

Reply:
the comment is wrong, I'll update it. This should just prevent an edge case and type script error when for some reason the studentExam.exam is not loaded or undefined